### PR TITLE
Allow exporting from stocks/ca/hcorr

### PR DESCRIPTION
**Pull request by alexferrari88**
# Description

The exporting function for stocks/ca/hcorr is documented but not implemented as reported by issue #1752. This PR solves that.

# How has this been tested?

`$> stocks/ca`
`$ stocks/ca> set XLE,XLK,XLY,XLI,XLP,XLC,XLV,XLF,XLU,XLB,XLRE`
`$ stocks/ca> hcorr --export png`

# Checklist:

- [X] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [X] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [X] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [X] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [X] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Review by DidierRLopes — APPROVED** [review]

**Comment by DidierRLopes** [comment]
Nice work @alexferrari88 !! 🦋

**Comment by DidierRLopes** [comment]
@alexferrari88 you just need to have linting tests to pass. You can do `black file path`

File: openbb_terminal/stocks/comparison_analysis/yahoo_finance_view.py [hunk 1]
```diff
@@ -226,5 +229,7 @@ def display_correlation(
 
     if not external_axes:
         theme.visualize_output()
-
+    
+    export_data(export, os.path.dirname(os.path.abspath(__file__)), "hcorr", df_similar)
```
Comment (piiq) [inline openbb_terminal/stocks/comparison_analysis/yahoo_finance_view.py:233]: @alexferrari88 most likely this line is longer than 88 chars and that's why black is complaining. Run `black .` from the project root and black will check and fix formatting in all the files of the project. 
Once you commit those changes it should be good

Comment (alexferrari88) [reply openbb_terminal/stocks/comparison_analysis/yahoo_finance_view.py:233]: thank you! strangely when I have run black it said that no files were unchanged...
anyway I have manually fixed it and it seems working (at least for me!)

**Comment by alexferrari88** [comment]
btw this is my first ever PR for a public project so forgive my naivete :)

**Comment by DidierRLopes** [comment]
> btw this is my first ever PR for a public project so forgive my naivete :)

We hope that this is the first of many 🤩

**Comment by jmaslek** [comment]
> btw this is my first ever PR for a public project so forgive my naivete :)

This project was my first PR on a public project as well :)

**Comment by piiq** [comment]
@alexferrari88 legend! 🚀

**Pull request merged**